omnibus/files/private-chef-ctl-commands/install.rb

```diff
@@ -31,6 +24,7 @@
     exit 1
   end
 
+  attributes_path = "#{base_path}/embedded/cookbooks/install_params.json"
```

Comment:
Same comment on log_level fatal. At this point I assume it's removal is purposeful.